pkg/forwarder/interceptor/interceptor.go

```diff
@@ -95,6 +104,14 @@ func (i *interceptor) CreateContainer(ctx context.Context, req *pb.CreateContain
 					logger.Printf("Failed to create dir: %v", err)
 				}
 			}
+
+			if isTargetPath(m.Source, volumeTargetPath) {
+				logger.Printf("Waiting for device mounted to: %s", m.Source)
+				err := waitForDeviceMounted(m.Source)
```

Comment:
I think `waitForDeviceMounted` should accept `ctx` as an argument, and should cancel waiting when `ctx` is cancelled.